### PR TITLE
[Feature] support query lake colocate table stable

### PR DESCRIPTION
**Pull request by starrocks-xupeng**
Signed-off-by: starrocks-xupeng <xupeng@starrocks.com>

## What type of PR is this：
- [ ] BugFix
- [x] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [ ] I have checked the version labels which the pr will be auto backported to target branch
  - [ ] 2.5
  - [ ] 2.4
  - [ ] 2.3
  - [ ] 2.2

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 2 / 8 (25.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/lake/StarOSAgent.java|0|6|00.00%|[484, 498, 501, 502, 503, 505]|
|:large_blue_circle:|com/starrocks/catalog/ColocateTableIndex.java|2|2|100.00%|[]|

**Review by wyb — APPROVED** [review]

File: fe/fe-core/src/main/java/com/starrocks/catalog/ColocateTableIndex.java [hunk 1]
```diff
@@ -635,7 +634,7 @@ public List<List<String>> getInfos() {
                 List<String> cols = groupSchema.getDistributionColTypes().stream().map(
                         e -> e.toSql()).collect(Collectors.toList());
                 info.add(Joiner.on(", ").join(cols));
-                info.add(String.valueOf(!unstableGroups.contains(groupId)));
+                info.add(String.valueOf(!isGroupUnstable(groupId)));
```
Comment (kevincai) [inline fe/fe-core/src/main/java/com/starrocks/catalog/ColocateTableIndex.java:637]: may need mark a TODO to cache the unstable result? queryStable API call may be costly.

**Review by kevincai — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

File: fe/fe-core/src/main/java/com/starrocks/catalog/ColocateTableIndex.java (hunk 1 above)
Comment (starrocks-xupeng) [reply fe/fe-core/src/main/java/com/starrocks/catalog/ColocateTableIndex.java:637]: in FE `stable` is persisted, i think starmgr should also do this

Comment (kevincai) [reply fe/fe-core/src/main/java/com/starrocks/catalog/ColocateTableIndex.java:637]: FE doesn't have multiple worker groups feature, the situation is different. Will consider how to cache the result if possible.

Comment (starrocks-xupeng) [reply fe/fe-core/src/main/java/com/starrocks/catalog/ColocateTableIndex.java:637]: better make application easier

**Comment by starrocks-xupeng** [comment]
run starrocks_admit_test

**Comment by starrocks-xupeng** [comment]
run starrocks_admit_test

**Comment by starrocks-xupeng** [comment]
run starrocks_admit_test

**Comment by starrocks-xupeng** [comment]
run starrocks_admit_test

**Comment by starrocks-xupeng** [comment]
run starrocks_admit_test

**Comment by starrocks-xupeng** [comment]
run starrocks_admit_test

**Comment by starrocks-xupeng** [comment]
run starrocks_admit_test

**Review by sduzh — APPROVED** [review]

**Pull request merged**